### PR TITLE
[doc] Agentic RAG Tutorial

### PR DESCRIPTION
**Pull request by davidmyriel**
Still need to add architecture and more details.
This tutorial is based on this example here: https://github.com/qdrant/examples/pull/56

[PREVIEW](https://deploy-preview-1314--condescending-goldwasser-91acf0.netlify.app/documentation/agentic-rag-crewai-zoom/)

### CODE REVIEW

**Review by kacperlukawski — CHANGES_REQUESTED** [review]
I left some inline comments. However, data upload part is missing in the corresponding PR to the examples, so I couldn't really test it.

File: qdrant-landing/content/documentation/agentic-rag-crewai-zoom.md [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+
+---
+
+## Usage
+
+### 1. Process Meeting Data
+The `data_loader.py` script processes meeting data and stores it in Qdrant:
+```bash
+python vector/data_loader.py
+```
```
[the remark this answers is not in the document]
Comment (thierrypdamiba) [reply qdrant-landing/content/documentation/agentic-rag-crewai-zoom.md]: Added links to the scripts. I also added information about what will happen after you run the commands.

File: qdrant-landing/content/documentation/agentic-rag-crewai-zoom.md [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+
+### 2. Launch the Interface
+Start the interactive app:
+```bash
+streamlit run vector/streamlit_app.py
+```
+
+### The Data Pipeline
+
+At the heart of our system is the data processing pipeline. We use a singleton pattern to ensure efficient resource usage:
```
[the remark this answers is not in the document]
Comment (thierrypdamiba) [reply qdrant-landing/content/documentation/agentic-rag-crewai-zoom.md:144]: Added clarification on why and how we are using the singleton pattern. Happy to take it out if it is unnecessary.

File: qdrant-landing/content/documentation/agentic-rag-crewai-zoom.md [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+# Agentic RAG With Claude, CrewAI & Qdrant 
+
+| Time: 45 min | Level: Beginner | Output: [GitHub](https://github.com/qdrant/examples/) |
+| --- | ----------- | ----------- |----------- |
+
+By combining the power of Qdrant for vector search and CrewAI for orchestrating modular agents, you can build systems that don't just answer questions but analyze, interpret, and act.
+
+Traditional RAG systems focus on fetching data and generating responses, but they lack the ability to reason deeply or handle multi-step processes. Agentic RAG solves this by combining:
+
+In this tutorial, we'll walk you through building an Agentic RAG system step by step. By the end, you'll have a working framework for storing data in a Qdrant Vector Database and extracting insights using CrewAI agents in conjunction with Vector Search over your data.
```
[the remark this answers is not in the document]
Comment (thierrypdamiba) [reply qdrant-landing/content/documentation/agentic-rag-crewai-zoom.md:17]: Added screenshots of the app and showing what is happening when you run the commands.

**Pull request merged**